public/scripts/tags.js

```diff
@@ -1620,8 +1655,8 @@ function onTagCreateClick() {
     toastr.success('Tag created', 'Create Tag');
 }
 
-function appendViewTagToList(list, tag, everything) {
-    const count = everything.filter(x => x == tag.id).length;
+function appendViewTagToList(list, tag, count) {
+    // const count = everything.filter(x => x == tag.id).length;
```

Comment:
I think you can just remove this line, no need to just comment it out. `count` is now just being passed in.